docs/Ordinal.md

```diff
@@ -1,13 +1,41 @@
 # Ordinal Support
 
-It is possible to use an ordinal as part of the collateral of a DLC.
-To do so, an [`OrdDescriptor`](../dlc-manager/src/contract/ord_descriptor.rs) must be used.
-This descriptor contains information about the ordinal being included in the DLC (location in the blockchain and the transaction that includes is), as well as information about the event upon which the DLC is based.
+This library now supports the locking of an ordinal as part of the collateral of a DLC.
+
+Notes:
+* An ordinal is inscribed into a single satoshi. However, that satoshi is often part of a larger UTXO. DLCs lock UTXOs, so please keep that context in mind.
+
+* In all cases, the gas fee to lock and later unlock the DLC is split between both parties.
+
+* For more information about DLCs, please see the main folder of this repo, the [`DLC-Link docs page`](https://docs.dlc.link), https://github.com/dlc-link, and the main https://github.com/discreetlogcontracts/dlcspecs specifications page.
+
+> Resizing UTXOs
+>
+> Please note, this package does not currently support the re-sizing of the UTXO containing the ordinal within the DLC. See [this section](#changing-postage) for more details
+
+
+## Supported types of DLCs
+
+### DLC with only the ordinal
+This package supports the use case when the offering party of the DLC locks the ordinal-containing UTXO into the DLC, and the counter-party (the acceptor) does not lock any collateral.
+
+In this case, the outcome is essentially binary, either the ordinal containing UTXO goes to the offeror, or the acceptor.
+
+### Ordinal with additional collateral
+This package also supports the case when the offering party of the DLC locks the ordinal-contianing UTXO into the dlc, AND any amount of addition collateral is locked into the DLC by either party.
+
+1. In this case, the outcome of the UTXO containing the ordinal will go to either party A or B, without resizing (as described earlier)
+2. The remaining collateral locked by either, or both, parties can be split between the parties based on the standard DLC enumerated or numerical outcome guidelines.
```

Comment:
I feel this could be summarized in a sentence saying that you can chose to attach an additional collateral or not. To me it's quite difficult to grasp as is.

Reply:
Well I was writing this from the perspective of someone who isn't fully versed in DLCs. So for that I was being more verbose and describing exactly what is happening in the DLC. Since there is no reference to Ordinals in the DLC specs anywhere, this can be used as the informational page. DLC Specs covers a lot of this high-level text. 

If you'd like however, I can shrink it back down, and find another place, perhaps on our dlc.link docs, to add this kind of less-technical review of ordinals and DLCs.

Reply:
I think it's important to have information about how the library can be used, but somehow I'm not seeing these sentences as very helpful. I think what would be most helpful would be to have some [examples](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#examples) if you feel motivated.

Reply:
That's cool. As I try out the library on our dlc.link system i'll see if I can add some examples